SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please report security issues to <email>
```

Comment:
You should set email as  [security@dotclear.net](mailto:security@dotclear.net) ;-)